### PR TITLE
correct demo site url

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@
 /web/bundles/
 /app/SymfonyRequirements.php
 /app/check.php
+
+/.idea/
```
Comment (dbu) [inline .gitignore]: this is specific to your dev environment and should be in your system git ignore, not in the project

**Review by dbu — COMMENTED** [review]
uh, thanks for spotting that one. this has been moved ages ago, seems we missed the links here.

can you please remove the change to the gitignore file?

File: .gitignore (hunk 1 above)
Comment (mmoll) [reply .gitignore]: there we go...

**Review by dbu — APPROVED** [review]
thank you!

**Pull request merged**